### PR TITLE
CU-29aq37e Populate data (application term, position, and subteam) taken from th…

### PR DESCRIPTION
**Pull request by IvanYu327**
…e application instance

### [CLICKUP TICKET LINK](https://app.clickup.com/t/r17re8)

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fill email template with data from emailData to populate application term and the subteam and position the applicant is applying for instead of using mockData

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by jennchenn** [comment]
Task linked: [CU-29aq37e Populate data taken from the application instance](https://app.clickup.com/t/29aq37e)

File: src/frontend/pages/recruitment/DecisionPage/DecisionPage.js [hunk 1]
```diff
@@ -166,9 +187,9 @@ const DecisionPage = () => {
     execEmail: 'john.doe@waterloop.ca',
     execPhoneNum: '(000) 000-0000',
     // Extracted from the decision portal table row.
-    position: 'Fullstack Developer',
-    subteam: 'Web',
-    nextTerm: 'Spring 2022',
+    // position: 'Fullstack Developer',
+    // subteam: 'Web',
+    // nextTerm: 'Spring 2022',
```
Comment (jmawloo) [inline src/frontend/pages/recruitment/DecisionPage/DecisionPage.js]: Don't need these lines now, can remove them :)

File: src/frontend/pages/recruitment/DecisionPage/DecisionPage.js [hunk 2]
```diff
@@ -151,6 +151,27 @@ const DecisionPage = () => {
     },
   ];
 
+  const formatTerm = (currentTerm) => {
+    const term = currentTerm?.slice(0,1)+currentTerm?.slice(1,-5).toLowerCase();
```
Comment (jmawloo) [inline src/frontend/pages/recruitment/DecisionPage/DecisionPage.js]: From my understanding currentTerm is represented as TERM-YEAR, so would be better if we could split the string by the '-' delimiter and then capitalize in order to account for all string lengths for TERM.

Also would be awesome if you could add move this function to utils.js :)

**Review by jmawloo — CHANGES_REQUESTED** [review]
Nice work, just a few comments :)

**Review by jmawloo — APPROVED** [review]

**Pull request merged**